### PR TITLE
CPUInfo: add DSP

### PR DESCRIPTION
**Pull request by meee1**
add some dsp function times to the output

### CODE REVIEW

**Comment by andyp1per** [comment]
Very cool! What results does it give you?

**Comment by meee1** [comment]
@andyp1per  sin and cos are faster, sqrt, same

File: Tools/CPUInfo/CPUInfo.cpp [hunk 1]
```diff
@@ -132,24 +136,31 @@ static void show_timings(void)
...
+
+    TIMEIT("sin()", v_out = sin(v_f), 100);
+    TIMEIT("cos()", v_out = cos(v_f), 100);
+    TIMEIT("tan()", v_out = tan(v_f), 100);
+    TIMEIT("acos()", v_out = acos(v_f * 0.2), 100);
+    TIMEIT("asin()", v_out = asin(v_f * 0.2), 100);
+    TIMEIT("atan2()", v_out = atan2(v_f * 0.2, v_f * 0.3), 100);
+    TIMEIT("sqrt()",v_out = sqrt(v_f), 100);
+    #if HAL_WITH_DSP && CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
+	TIMEIT("arm_sqrt_f32()", arm_sqrt_f32(v_f, (float32_t*)&v_out), 100);
```
Comment (khancyr) [inline Tools/CPUInfo/CPUInfo.cpp:159]: small indent issue there

**Review by khancyr — APPROVED** [review]
LGTM

No risk merging this

**Pull request merged**